style.css

```diff
@@ -0,0 +1,67 @@
+body {
+  min-width: 300px;
+  background-color: white;
+  font-family: sans-serif;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  top: 15px;
+}
+
+#navbar {
+  position: fixed;
+  min-width: 260px;
+  display: block;
+  border-right: solid;
+  height: 100vh;
+  margin-left: 10px;
+}
+
+.hero {
+  border-bottom: 1px solid black;
+  font-size: 25px;
+  color: darkblue;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+div {
```

Comment:
I would suggest you give this div a class so as to make your output looks nice as seen in the real project output

Reply:
- [ ] Another great recommendation, but the 'why' seems lacking again. Here you can tell the reviewee how adding CSS to generic tags will affect the `div` tags in the entire document, and how adding a class will make the CSS stick to only this one component. Additionally, you can share a link to a documentation as an example to better explain your point :)